src/app/question-points/question-points.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-question-points',
+  template: `
+    <div>
+    {{'Number of questions remaining:' + NumQuestions}} 
+    </div>
+  `,
+  styles: [
+  ]
+})
+export class QuestionPointsComponent implements OnInit {
+  @Input() NumQuestions: Number;
+
+  constructor() {this.NumQuestions = 0} // should I initialize it ? 
```

Comment:
Yeah, well.
When I created the angular app, I activated strict checks. So the code forces the programmer to explicitely state the value may be uninitialized. 
You have a good explanation here: https://stackoverflow.com/a/60113028